src/Microsoft.OData.Core/Microsoft.OData.Core.cs

```diff
@@ -7,8 +7,7 @@
 // </copyright>

```

Comment:
I'll have to modify the T4 template in order to do that. These files should not be edited by hand.

I can do that if you think it's really important.